Sources/Apodini/Configurations/Application/Application+Vapor.swift

```diff
@@ -13,15 +13,21 @@ extension Vapor.Application {
         var app: Vapor.Application
 
         func didBoot(_ application: Application) throws {
-            try app.start()
+            if let address = application.http.address {
+                try app.server.start(address: Vapor.BindAddress(from: address))
+            } else {
+                try app.server.start()
+            }
+            try app.boot()
```

Comment:
Are you sure this is correct? Executing `app.boot()` in a function called `didBoot` just smells like there is something wrong. I haven't looked into the details of Vapor boot-up processes, though.

Reply:
Did boot relates to the Apodini application. So we boot up the vapor application after the Apodini application did boot.

Reply:
Ah, I see...thanks for the explanation.